src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
Same as for width, avoid using fixed height. You should let your content define the height of your block.